enhancements/windows-containers/health-management.md

```diff
@@ -168,6 +176,11 @@ When making use of a PowerShell script, retries should be present within the scr
 is waiting for the HNS network to be created in order to create an HNS endpoint. In that specific case, the PowerShell
 script which creates the endpoint should include a retry period which waits for the needed Network to exist.
 
+For ease of processing, services with the bootstrap flag set to true should have a higher priorities than all 
+non-bootstrap services. 
```

Comment:
Please add the WICD will throw an error if this is not the case.

Reply:
Missed this earlier, I don't think this will have a noticeable effect on ease of processing. Its an unnecessary restriction that id prefer to drop.